python/grass/script/setup.py

```diff
@@ -24,20 +24,20 @@
     # query GRASS itself for its GISBASE
     # (with fixes for specific platforms)
     # needs to be edited by the user
-    grass7bin = 'grass79'
+    grass8bin = 'grass80'
     if sys.platform.startswith('win'):
         # MS Windows
-        grass7bin = r'C:\OSGeo4W\bin\grass79.bat'
+        grass8bin = r'C:\OSGeo4W\bin\grass80.bat'
         # uncomment when using standalone WinGRASS installer
-        # grass7bin = r'C:\Program Files (x86)\GRASS GIS 7.9.0\grass79.bat'
+        # grass8bin = r'C:\Program Files (x86)\GRASS GIS 8.0.0\grass80.bat'
```

Comment:
do we provide still a 32 bit standalone installer C:\Program Files (x86)? @landam